server/src/main/resources/reference-docker.conf

```diff
@@ -9,6 +9,11 @@ modeldb: {
   metadataDb: {
     "host" : "MONGODB_HOST",
     "port" : 27017,
+
+    "//": "If you want a username and password"
```

Comment:
This isn't JSON, it's [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md). Because of this, you should use regular comments:
```
# If you want a username and password for mongo:
# username: "my_username",
# password: "my_password",
```
You should also update the non-docker config files to have these new fields.

Reply:
fixed the commenting format